.github/workflows/node-ci.yml

```diff
@@ -1,5 +1,9 @@
 on:
   workflow_call:
+    secrets:
```

Comment:
It might be cleaner to use {{ github.token }} and avoid passing this down?

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context:~:text=example%2C%20ffac537e6cbbf934b08745a378932722df287a53.-,github.token,-string

Reply:
Would this work in a cross-org context @PatrickDinh @cuzzlor ?

Reply:
Good point
If not maybe it's the default if the secret isn't passed